pkgs/apps/meep/default.nix

```diff
@@ -0,0 +1,103 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub, autoreconfHook, pkg-config
+, gfortran, mpi, blas, lapack, fftw, hdf5-full, swig, gsl, harminv, libctl
+, libGDSII, openssh, guile
+# Python
+, python, numpy, scipy, matplotlib, h5py, cython, autograd, mpi4py
+} :
+
+buildPythonPackage rec {
+  pname = "meep";
+  version = "1.18.0";
+
+  src = fetchFromGitHub {
+    owner = "NanoComp";
+    repo = pname;
+    rev = "v${version}";
+    sha256= "08l4mczkh08dp90c4dlnyx6lsg093li0xqnnbh7q8k363cr8lx81";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    gfortran
+    pkg-config
+    swig
+    mpi
```

Comment:
It should be enough to have `mpi` in `propagatedBuildInputs`.

Reply:
Interestingly it is not. I'm not sure why, but `configure` does not find it, if only in `propagatedBuildInputs`